ElectronClient/package-lock.json

```diff
@@ -10047,11 +10047,6 @@
         "through2": "^2.0.3"
       }
```

Comment:
If package.json hasn't change, this file should not be included so please remove it.

Reply:
I believe this is from a merge commit which had a new dependency for `aws-sdk` package. But I rebased on the latest commit on master so that I wouldn't need to commit his file, just to be sure..